BlueprintUI/Sources/Internal/ElementIdentifier.swift

```diff
@@ -41,24 +41,24 @@
 
  You will note that the identifiers remain stable, which ultimately ensures that views are reused.
  */
-struct ElementIdentifier: Hashable, CustomDebugStringConvertible {
+struct ElementIdentifier: Hashable, CustomStringConvertible {
 
-    let elementType: ObjectIdentifier
+    let elementType: Metatype
     let key: AnyHashable?
 
     let count: Int
 
     init(elementType: Element.Type, key: AnyHashable?, count: Int) {
 
-        self.elementType = ObjectIdentifier(elementType)
+        self.elementType = Metatype(elementType)
```

Comment:
Changed because it was a cheaper or more legible?

Reply:
> Changed because it was a cheaper or more legible?

The latter I suppose. `Any.Type` is annoying to work with because it's not equatable or hashable, and `ObjectIdentifier` has a useless `description`.